PowerShellTools/PowerShellToolsPackage.cs

```diff
@@ -284,13 +284,14 @@ protected override void Initialize()
             }
         }
 
-        private void InitializeInternal()
+        private async void InitializeInternal()

```

Comment:
Better use async Task here, then await the async method in the calling method. Besides, append Async to the method name as it's convention to have an async method ending with Async.